parser_tool/static/js/src/quick_lemma.js

```diff
@@ -0,0 +1,108 @@
+(function($) {
+  const ApiClient = window.app.ApiClient;
+    // practice
+    async function sorted_lemmas() {
```

Comment:
Note that since `jqXhr.then(...)` is being used to handle the response, it's not strictly necessary to use the async/await syntax here, so it could be removed without doing any harm.